src/vs/workbench/contrib/notebook/browser/view/output/outputRenderer.ts

```diff
@@ -16,28 +16,34 @@ export class OutputRenderer {
 	private readonly _richMimeTypeRenderers = new Map<string, IOutputTransformContribution>();
 
 	constructor(
-		notebookEditor: ICommonNotebookEditor,
-		instantiationService: IInstantiationService
+		private readonly notebookEditor: ICommonNotebookEditor,
+		private readonly instantiationService: IInstantiationService
 	) {
-		for (const desc of OutputRendererRegistry.getOutputTransformContributions()) {
```

Comment:
Building the list of renderers in the ctor will not work, as that's too early in the life cycle & other built-in renderers have not yet been regsitered.
We need to lazy load them (giving time for them to get registered)

Reply:
Other than postponing when `OutputRendererRegistry.getOutputTransformContributions` is called (which would never be correct now I think about is, as code renderers can register any time). We may want to move to the logic into `OutputRendererRegistry`.